README.md

```diff
@@ -241,7 +241,7 @@ various products. These incremental builds are a big timesaver when developing
 and debugging.
 
     cd ${SWIFT_BUILD_DIR}
-    ninja swift
+    ninja compiler
```

Comment:
Hmm, this was the recommendation I heard, but it still does some work on null builds (maybe it is a phony target?). We could get rid of the `swift-frontend` below and use `swift-frontend` here.

Reply:
Gotcha! I've updated the branch appropriately.

Reply:
Thanks!